baker/roster.json

```diff
@@ -132,6 +132,10 @@
             "materials": [
                 {
                     "type": "cv"
+                },
+                {
+                    "type": "video",
+                    "url": "https://www.youtube.com/watch?v=h8nibkvCBiY"
```

Comment:
Heya @DesignedByQ please could you update to have just the video ID as in Lily's request? I'll tag you in that post with the example now :)